src/app/screens/BookList/layout.js

```diff
@@ -0,0 +1,40 @@
+import React, { PureComponent } from 'react';
+import { FlatList, Text } from 'react-native';
+
+import Book from './components/Book';
+import styles from './styles';
+
+import WithLoading from '../../components/WithLoading';
+import { NO_BOOKS } from '../../../constants/errors';
+
+export const generateBookUri = book => `https${book.image.url.substring(4)}?id=${book.id}`
```

Comment:
⚠️ Do you think this is the right place to put this function declaration? Maybe in an utiles file?